src/codeu/chat/server/MessagePersist.java

```diff
@@ -0,0 +1,182 @@
+
+package codeu.chat.server;
+
+import java.sql.*;
+import java.util.Collection;
+
+import codeu.chat.common.BasicController;
+import codeu.chat.common.Conversation;
+import codeu.chat.common.Message;
+import codeu.chat.common.RawController;
+import codeu.chat.common.User;
+import codeu.chat.util.Logger;
+import codeu.chat.util.Time;
+import codeu.chat.util.Uuid;
+
+public final class MessagePersist {

```

Comment:
Is there a better name for this class? Proper naming can really help the readability and maintainability of code for yourself and team members.

For example, you currently do something like:
```
MessagePersist msgData = new MessagePersist();
msgData.createUserTable(...);
msgData.retrieveUser(...);
msgData.insertUserTable(...);
```
One possible improvement could be:
```
Database db = new Database();
db.createUserTable(...);
db.initializeUsers(...);
db.insertUser(...);
```
Thoughts? You are of course free to disagree or come up with a better convention :)